### PR TITLE
Fixed tests

### PR DESCRIPTION
**Pull request by mbilozub**
Fixed some tests, removed extra spaces from altitude string on the osd screen.

### CODE REVIEW

**Comment by nerdCopter** [comment]
* i'd like to approve, since it does not impact flight control; however,
* i dont understand the `targetPidLooptime = 63;` vs `62` which was introduced by Helio.
* test pidloop vs flightware pidloop should be considered
* if this can be approved, then #280 can be approved.

**Comment by mbilozub** [comment]
> * i'd like to approve, since it does not impact flight control; however,
> * i dont understand the `targetPidLooptime = 63;` vs `62` which was introduced by Helio.
> * test pidloop vs flightware pidloop should be considered
> * if this can be approved, then #280 can be approved.

The target PID loop time for 16kHz is 62.5, which is rounded to 63 instead of 62. I found a fix here - 
targetPidLooptime

**Comment by nerdCopter** [comment]
perfect, thank you @mbilozub !

File: src/test/unit/blackbox_unittest.cc [hunk 1]
```diff
@@ -104,7 +104,7 @@ TEST(BlackboxTest, TestInitIntervals)
     EXPECT_EQ(65536, blackboxSInterval);
 
     // 16kHz PIDloop
-    targetPidLooptime = 62;
+    targetPidLooptime = 63;
```
[the remark this answers is not in the document]
Comment (mbilozub) [reply src/test/unit/blackbox_unittest.cc:107]: These changes will be made in this [commit](https://github.com/emuflight/EmuFlight/commit/03d068b17d0f33270768db4d14c724b12c83d7e9) but not changed in the unit tests.

**Pull request merged**